### ISSUE TITLE
Production ready ?

### ISSUE THREAD

**Issue by kirantpatil** [ramontayag/storey#56]
Hi,

I would like to know if it is in production ?

How easy it is compared to Apartment in terms of understanding and operationally ?

Thanks.

**Comment by ramontayag** [comment]
Yes I use this in production. Apartment seems more up to date though. Check the commit history 🙂 
     
     
      
     
    
    
     
     
       -- 
      Sent from Canary 
      
      
       
      
     
    
    
    
     
     
       On Wednesday, Oct 12, 2022 at 1:40 PM, kirantpatil ***@***.***> wrote: 
       
      
      
       
      Hi, 
      I would like to know if it is in production ? 
      How easy it is compared to Apartment in terms of understanding and operationally ? 
      Thanks. 
      —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***> 
      ***@***.***": ***@***.***": "EmailMessage","potentialAction": ***@***.***": "ViewAction","target": "https://github.com/ramontayag/storey/issues/56","url": "https://github.com/ramontayag/storey/issues/56","name": "View Issue"},"description": "View this Issue on GitHub","publisher": ***@***.***": "Organization","name": "GitHub","url": "https://github.com"}}]

**Comment by kirantpatil** [comment]
Thank you.

During my research I found [Postgres+Citus ](https://github.com/citusdata/activerecord-multi-tenant/) which is opensource. 

Have you had a chance to look in to it. Microsoft made Citus extension opensource ?

**Comment by ramontayag** [comment]
I haven't looked at citus in a long time -- since it was closed source.
